### PR TITLE
cmd: pretend we're running on Ubuntu in TestExecInCoreSnapUnsetsDidRe…

### PR DESCRIPTION
**Pull request by bboozzoo**
…exec()

Mock distribution os-release so that the test runs in a fixed environment.

### CODE REVIEW

File: cmd/cmd_test.go [hunk 1]
```diff
@@ -313,6 +313,13 @@ func (s *cmdSuite) TestExecInCoreSnapUnsetsDidReexec(c *C) {
 	os.Setenv("SNAP_DID_REEXEC", "1")
 	defer os.Unsetenv("SNAP_DID_REEXEC")
 
+	restore := release.MockReleaseInfo(&release.OS{ID: "ubuntu"})
+	defer func() {
+		restore()
+		dirs.SetRootDir(s.fakeroot)
```
Comment (zyga) [inline cmd/cmd_test.go]: Did you mean `dirs.SetRootDir("/")`?

Comment (bboozzoo) [reply cmd/cmd_test.go]: No, I actually meant to restore it to whatever was set up by the test suite right here: https://github.com/snapcore/snapd/blob/master/cmd/cmd_test.go#L62

Mocking release info slightly alters how the directories are set up, once it's restored SetRootDir() should also restore the directory paths to their original values.

Comment (zyga) [reply cmd/cmd_test.go]: +1

**Comment by mvo5** [comment]
Thanks for chasing this!

**Comment by bboozzoo** [comment]
Travis hit a timeout, restarted the build.

**Pull request merged**